packages/apex-parser-ast/src/types/symbol.ts

```diff
@@ -396,11 +392,11 @@ export type SymbolResolutionStrategy =
 
 export type Range = {
   startLine: number;
-  /** Start column (1-based) */
+  /** Start column (0-based) */
   startColumn: number;
   /** End line (1-based) */
```

Comment:
that start is 0 based but the end is 1 based?

Reply:
positions/ranges in LSP requests are 0-based indexes for lines and columns
positions/ranges in antlr are 1-based indexes for lines and 0-based indexes columns